src/loadConfig.js

```diff
@@ -0,0 +1,31 @@
+const path = require('path')
+const fs = require('fs')
+const YAML = require('yaml')
+
+const CONFIG_FILE_OVERRIDE = 'custom/config.yaml'
+const CONFIG_FILE = 'config.yaml'
```

Comment:
I was thinking it's better to give it a name more clear that it shouldn't be edited by hand.
What about `defaultConfig.yaml`?

Also, we should have a warning at the start of that file stating so.

Reply:
My thinking was, that is nice to keep everything related to the custom fork inside `custom/`
The fork steps would guide you how to create the config

> What about defaultConfig.yaml?

Yep, good idea. What about `config-default.yaml`? or `default-config.yaml`

Reply:
I'm fine with either. Just trying to drive the point that this shouldn't be edited if people want to keep the fork compatible.